chaos-monkey-spring-boot/src/main/java/de/codecentric/spring/boot/chaos/monkey/configuration/AssaultProperties.java

```diff
@@ -56,14 +60,15 @@
 
   private double memoryFillTargetFraction = 0.25;
 
+  private String memoryCronExpression;
+
   private volatile boolean cpuActive = false;
 
   private int cpuMillisecondsHoldLoad = 90000;
 
   private double cpuLoadTargetFraction = 0.9;
 
-  @Value("${chaos.monkey.assaults.runtime.scope.assault.cron.expression:OFF}")
-  private String runtimeAssaultCronExpression;
```

Comment:
I think we should keep the property and make it deprecated and give people time to stop using it and eventually remove it in the next version.

Until we deprecate it we can use the individual expressions to override the runtimeAssaultCronExpressions. Eg. if you define a global one, you can still override it on cpuCronExpression level. 

But there most likely also easier alternatives